interface/usergroup/usergroup_admin_add.php

```diff
@@ -237,7 +237,17 @@ function authorized_clicked() {
 <tr>
```

Comment:
also should do in the password change script at:
interface/usergroup/user_info.php

And would do in way to not duplicate code. Could add checkPasswordStrength function to the library/js/utility.js script. And can convert those lines with php (since will be in a javascript library file) to instead use the openemr javascript translation function, for example:
```
 document.getElementById('password_strength_text').innerText = xl('Very Weak');
```

Reply:
Added the file to utility.js in the latest commit. 
It is working in both places now. 

<img width="500" alt="image" src="https://github.com/openemr/openemr/assets/43812511/3f02497e-0c57-4bc0-94ec-ac9f03c18ab8">